src/openrct2-ui/windows/RideList.cpp

```diff
@@ -686,7 +686,7 @@ static void window_ride_list_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi,
                 break;
             case INFORMATION_TYPE_RUNNING_COST:
                 formatSecondary = STR_RIDE_LIST_RUNNING_COST_UNKNOWN;
-                if (ride->upkeep_cost != (money16)(uint16_t)0xFFFF)
+                if (ride->upkeep_cost != static_cast<money16>(static_cast<uint16_t>(0xFFFF)))
```

Comment:
Does it need the double cast here?

Reply:
[Seems like it doesn't](https://godbolt.org/z/JGcB2o)

Reply:
Please provide a suggestion comment, I can then merge it

Reply:
```suggestion
                if (ride->upkeep_cost != static_cast<money16>(0xFFFF))
```